openstudiocore/src/model/SiteGroundTemperatureDeep_Impl.hpp

```diff
@@ -0,0 +1,198 @@
+/**********************************************************************
+ *  Copyright (c) 2008-2016, Alliance for Sustainable Energy.
+ *  All rights reserved.
+ *
+ *  This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU Lesser General Public
+ *  License as published by the Free Software Foundation; either
+ *  version 2.1 of the License, or (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *  Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public
+ *  License along with this library; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ **********************************************************************/
+
+#ifndef MODEL_SITEGROUNDTEMPERATUREDEEP_IMPL_HPP
+#define MODEL_SITEGROUNDTEMPERATUREDEEP_IMPL_HPP
+
+#include "ModelAPI.hpp"
+#include "ModelObject_Impl.hpp"
+
+namespace openstudio {
+namespace model {
+
+namespace detail {
+
+  /** SiteGroundTemperatureDeep_Impl is a ModelObject_Impl that is the implementation class for SiteGroundTemperatureDeep.*/
+  class MODEL_API SiteGroundTemperatureDeep_Impl : public ModelObject_Impl {
+    Q_OBJECT;
```

Comment:
We do not want to use Q_OBJECT and Q_PROPERTY for new objects as we are reducing our dependence on Qt